acm_alerts/main.tf

```diff
@@ -0,0 +1,22 @@
+data "terraform_remote_state" "vpc" {
+  backend = "s3"
+
+  config = {
+    bucket = var.remote_state_bucket_name
+    key    = "vpc/terraform.tfstate"
+    region = var.region
+  }
+}
+
+# ssm parameter
+data "aws_ssm_parameter" "ssm_token" {
+  name = var.acm_alerts_info["slack_ssm_token"]
+}
+
+locals {
+  acm_alerts_info = var.acm_alerts_info
+  function_name   = "acm_alerts_handler"
```

Comment:
Will this cause conflicts in delius-core-dev/delius-core-sandpit - should it be prefixed with the environment name?

Reply:
added prefix: environment_name